src/core/components/courses/ProfileTraineeFollowUp.vue

```diff
@@ -29,8 +29,8 @@
     <div v-if="questionnaireActivities.length" class="q-my-xl">
       <p class="text-weight-bold">Réponses aux questionnaires</p>
       <div class="flex row">
-        <div v-for="activity in questionnaireActivities" :key="activity._id" class="card-container">
-          <q-card class="q-ma-sm q-py-sm card">
+        <div v-for="activity in questionnaireActivities" :key="activity._id" class="q-ma-sm card-container">
+          <q-card class="q-py-sm fit">
```

Comment:
A quoi sert le fit ? Quelle est la difference entre q-py-sm et q-pb-sm ?

Reply:
le fit sert a fixer la height et la width a 100%, le q-py-sm ajoute un padding sur l'axe y (au dessus et en dessous de la div), alors que le q-pb-sm le ferra juste au bottom..
ton retour me fait remarquer que je peux simplifier encore, comme j'ai une div card-container en v-for avec une q-card dedans, je peux tout simplement enlever le fit sur la q-card et appliquer le v-for dirrectement sur la q-card